forem/forem
# Set default box-sizing

## What type of PR is this? (check all applicable)


## Description

I think we're finally ready to get rid of that crayons-specific box-sizing properties... I've clicked through most of the common views and could find any broken things.. But another pair of eyes would be definitely appreciated.

## QA Instructions, Screenshots, Recordings

You gotta click through everything/anything and see if something is broken (e.g. have more padding)..

### UI accessibility concerns?

Nope.

## Added/updated tests?

Not needed.

## [Forem core team only] How will this change be communicated?

It won't.
